packages/components/pie-tag/src/index.ts

```diff
@@ -90,6 +73,33 @@ export class PieTag extends LitElement implements TagProps {
         </button>`;
     }
 
+    render () {
+        const {
+            disabled,
+            isInteractive,
+            isStrong,
+            size,
+            variant,
+            iconPlacement,
+        } = this;
+
+        const classes = {
+            'c-tag': true,
+            [`c-tag--${size}`]: true,
+            [`c-tag--${variant}`]: true,
+            'c-tag--disabled': disabled,
+            'c-tag--strong': isStrong,
+            'c-tag--interactive': isInteractive,
+            [`c-tag--icon-placement--${iconPlacement}`]: isInteractive && iconPlacement,
```

Comment:
this is the main change, the rest is a result of moving the handlers to the top.